pkg/dataloader/types.go

```diff
@@ -66,3 +71,37 @@ func WriteDataFile(filename string, dataFile DataFile) error {
 	}
 	return nil
 }
+
+// WriteDurations writes multiple duration metrics to a file in the autodol format.
```

Comment:
typo autodol -> autodl

I feel like this function would fit better under [e2eanalysis](https://github.com/jianlinliu/origin/tree/4efc9c6086764c40a5f4e942ac0678d60c108e16/pkg/e2eanalysis) either as a separate `util` or in `e2eanalysis.go`

Reply:
Sure. Moved it to `e2eanalysis.go`